config.toml

```diff
@@ -12,43 +12,30 @@
    StartInEpochEnabled = true
 
    # SCDeployEnableEpoch represents the epoch when the deployment of smart contracts will be enabled
-   SCDeployEnableEpoch = 6
+   SCDeployEnableEpoch = 1000000
 
    # BuiltInFunctionsEnableEpoch represents the epoch when the built in functions will be enabled
-   BuiltInFunctionsEnableEpoch = 116
+   BuiltInFunctionsEnableEpoch = 1000000
 
    # RelayedTransactionsEnableEpoch represents the epoch when the relayed transactions will be enabled
-   RelayedTransactionsEnableEpoch = 116
+   RelayedTransactionsEnableEpoch = 1000000
 
    # PenalizedTooMuchGasEnableEpoch represents the epoch when the penalization for using too much gas will be enabled
-   PenalizedTooMuchGasEnableEpoch = 6
+   PenalizedTooMuchGasEnableEpoch = 0
 
    # SwitchJailWaitingEnableEpoch represents the epoch when the system smart contract processing at end of epoch is enabled
-   SwitchJailWaitingEnableEpoch = 6
+   SwitchJailWaitingEnableEpoch = 0
 
    # BelowSignedThresholdEnableEpoch represents the epoch when the change for computing rating for validators below signed rating is enabled
-   BelowSignedThresholdEnableEpoch = 6
+   BelowSignedThresholdEnableEpoch = 0
 
    # SwitchHysteresisForMinNodesEnableEpoch represents the epoch when the system smart contract changes its config to consider
    # also (minimum) hysteresis nodes for the minimum number of nodes
-   SwitchHysteresisForMinNodesEnableEpoch = 6
-
-    # TransactionSignedWithTxHashEnableEpoch represents the epoch when the node will also accept transactions that are
-    # signed with the hash of transaction
-    TransactionSignedWithTxHashEnableEpoch = 116
-
-   # MetaProtectionEnableEpoch represents the epoch when the transactions to the metachain are checked to have enough gas
-   MetaProtectionEnableEpoch = 116
-
-   # AheadOfTimeGasUsageEnableEpoch represents the epoch when the cost of smart contract prepare changes from compiler per byte to ahead of time prepare per byte
-   AheadOfTimeGasUsageEnableEpoch = 116
+   SwitchHysteresisForMinNodesEnableEpoch = 0
```

Comment:
I do not think this is ok. It should be at least 1 epoch as the systemSCs.go is not called in the first epoch

Reply:
good catch! Thanks.